dartagnan/src/test/java/com/dat3m/dartagnan/comparison/C11AndOpenCLTest.java

```diff
@@ -0,0 +1,43 @@
+package com.dat3m.dartagnan.comparison;
+
+import com.dat3m.dartagnan.configuration.Arch;
+import com.dat3m.dartagnan.utils.rules.Provider;
+import com.dat3m.dartagnan.utils.rules.Providers;
+import com.dat3m.dartagnan.wmm.Wmm;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.io.IOException;
+
+@RunWith(Parameterized.class)
+public class C11AndOpenCLTest extends AbstractComparisonTest {
```

Comment:
You should also add a test that compares these test for races. You can simply create a new class that extends this one and override the propertyProvider method (assuming `LitmusOpenClRacesTest` does not run on litmus/OPENCL/portedFromC11/)

Reply:
Added